src/Octoshift/GithubApi.cs

```diff
@@ -581,5 +560,51 @@ public virtual async Task<MannequinReclaimResult> ReclaimMannequin(string orgId,
 
             return data.ToObject<MannequinReclaimResult>();
         }
+
+        private static object GetMannequinsPayload(string orgId)
+        {
+            var query = "query($id: ID!, $first: Int, $after: String)";
+            var gql = @"
+                node(id: $id) {
+                    ... on Organization {
+                        mannequins(first: $first, after: $after) {
+                            pageInfo {
+                                endCursor
+                                hasNextPage
+                            }
+                            nodes {
+                                login
+                                id
+                                claimant {
+                                    login
+                                    id
+                                }
+                            }
+                        }
+                    }
+                }";
+
+            return new
+            {
+                query = $"{query} {{ {gql} }}",
+                variables = new { id = orgId }
+            };
+        }
+
+        private static Mannequin MannequinBuilder(JToken mannequin)
+        {
+            return new Mannequin
+            {
+                Id = (string)mannequin["id"],
+                Login = (string)mannequin["login"],
+                MappedUser = mannequin["claimant"].Any()
+                                    ? new Claimant
+                                    {
+                                        Id = (string)mannequin["claimant"]["id"],
+                                        Login = (string)mannequin["claimant"]["login"]
+                                    }
+                                    : null
+            };
+        }
```

Comment:
Again based on my suggestion there will be no need to create these private methods.